<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesWindow</class>
 <widget class="QMainWindow" name="NotesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOK">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>530</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbItalic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Italic ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNoItalic">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>5</y>
      <width>86</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Italic off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCancel">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>530</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbBoldOn">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>5</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bold ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbBoldOff">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>5</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Bold off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUnderlineOn">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>5</y>
      <width>86</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUnderlineOff">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>5</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Underline off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbVerse">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>5</y>
      <width>116</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Add Verse Ref</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
